library.json

```diff
@@ -14,7 +14,7 @@
     {
       "name": "ESPAsyncUDP",
       "authors": "Hristo Gochkov",
-      "platforms": ["espressif", "espresif8266"]
```

Comment:
This should be `"platforms": ["espressif32", "espresif8266"]` as it is listed on https://docs.platformio.org/en/latest/platforms/espressif32.html

May you test if it compiles ok?

Reply:
No it compiles not ok. see thread for comment.